NEWS

```diff
@@ -11,9 +11,98 @@
 ### Features:
 ### Bugfixes:
 
-## 1.17.0-rc1 (April 19)
-TBD
-
+## 1.17.0-rc1 (May 9, 2024)
+### Features:
+#### UCP
+* Improved performance calculation for rendezvous protocol
+* Enabled short protocol for non-host memory types on empty messages
+* Improved performance estimation for empty messages by removing non-relevant overheads
+* Added RMA_ZCOPY_MAX_SEG_SIZE configuration parameter to allow modifying segment size for RMA-ZCOPY protocols
+* Added support for separate intra/inter-node rendezvous thresholds
+* Added support for minimal fragment size in rendezvous protocol
+* Added support for protocol variants
+* Added support for resetting request during send operation
+* Added UCX_PROTO_OVERHEAD configuration variable to allow setting protocol overheads
+* Improved performance for combined Active Message/RMA scenarios by separating them to different lanes
+* Added support for device staging buffers in pipeline protocols 
+#### RDMA CORE (IB, ROCE, etc.)
+* Added UCX_REVERSE_SL env var to allow configuring reverse SL and set default to UCX_IB_SL value
+* Added support for GID auto-detection in Floating LID based routing
+* Imporved performance of RC transport when hardware tag matching is not available
+* Added support for multithreading KSM registration of unaligned buffers
+* Added IB_SEND_OVERHEAD and MM_[SEND|RECV]_OVERHEAD configuration variables
+#### GPU (CUDA, ROCM)
+* Added support for oneAPI Level-Zero library for intel GPUs
+#### UCS
+* Added support for rcache dynamic region alignment
+* Added dynamic bitmap data structure
+* Added support for advanced key-value parsing for UCX configuration
+* Added piecewise linear function data structure
+* Added support for allocating dynamic arrays on stack
+#### Tests
+* Added wire compatibility tests for v1.16 
+#### Tools
+* Added support for device memory allocation in UCX perftest
+* Added a script to use for squashing commits after PR approval
+* Added support for DPU cross-gvmi daemon in UCX perftest
+#### Java
+* Added support for EP local socket address API in JUCX
+#### Build
+* Added Address Sanitizer support
+* Added a helper shell script to run static checks
+#### AZP
+* Added Address Sanitizer job to CI
+* Disabled re-run of all tests when Valgrind tests fail
```

Comment:
Replaced Valgrind tests by Address Sanitizer

Reply:
it was done in steps, but currently valgrind is completely disabled: https://github.com/openucx/ucx/blob/eae754ee3be8865a6ba37008efe84ce8f572623b/buildlib/pr/tests.yml#L10

Reply:
and if so also removes line 100:
`* Increased timeout of Valgrind tests`
as it is not relevant anymore

Reply:
> and if so also removes line 100: `* Increased timeout of Valgrind tests` as it is not relevant anymore

Right, should remove it as well

Reply:
also should remove line 55:
`* Added Address Sanitizer job to CI`